### PR TITLE
Autokeras hyperparam tuning

### PR DESCRIPTION
**Pull request by babenek**
## Description

Please include a summary of the change and which is fixed.

- Add kerastuner package to find better parameters. Fixes #499, #508, #517
- Apply for dropout layers threshold and retrained
- Refactored ml features with normalized inputs and optimization (entropies in one method)

## How has this been tested?

Please describe the tests that you ran to verify your changes.

- [x] UnitTest
- [x] Benchmark

### CODE REVIEW

**Review by Yullia — DISMISSED** [review]
Approved

**Review by xDizzix — APPROVED** [review]

**Review by iuriimet — APPROVED** [review]
